src/common/ErrorPage/styled.js

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+import { ReactComponent as DangerIcon } from "../../images/danger.svg";
+
+export const Wrapper = styled.div`
+    max-width: 592px;
+    width: 100%;
+    margin: 195px auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px) {
+        margin-top: 120px;
+    }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        margin-top: 50px;
+    }
+`;
+
+export const StyledDangerIcon = styled(DangerIcon)`
+    width: 120px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        width: 80px;
+    }
+`;
+
+export const Title = styled.h1`
+    text-align: center;
+    font-weight: 600;
+    font-size: 35px;
+    line-height: 120%;
+    margin: 38.16px 0 0;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        font-size: 18px;
+        margin-top: 0;
+    }
+`;
+
+export const Text = styled.p`
+    text-align: center;
+    margin: 24px 82.73px;
+    padding: 0;
```

Comment:
`padding: 0;` is superfluous. What is 82pxs side margin doing?

Reply:
right

Reply:
fixed that